src/language/en.json

```
@@ -1,8 +1,10 @@
 {
+    "name": "English",
```

```suggestion
    "languageName": "English",
```
to specify a bit more. Name for me is a persons name. Or maybe just use "language".